app/bot.go

```diff
@@ -93,6 +92,12 @@ func (a *App) getOrCreateWarnMetricsBot(botDef *model.Bot) (*model.Bot, *model.A
 				return nil, model.NewAppError("getOrCreateWarnMetricsBot", "app.bot.createbot.internal_error", nil, nErr.Error(), http.StatusInternalServerError)
 			}
 		}
+
+		if err != nil {
+			mlog.Error(err.Error())
```

Comment:
Gentle reminder to check our logging [principles](https://docs.google.com/document/d/160-9Qfk-1a2_gOU-VHfZ5CbUdkY8BZxq-2VL7XYwqGA) before merging this change.